bmcapi/go.mod

```diff
@@ -1,12 +1,7 @@
 module github.com/phoenixnap/go-sdk-bmc/bmcapi
 
-go 1.17
-
-require golang.org/x/oauth2 v0.0.0-20210218202405-ba52d332ba99
+go 1.13
```

Comment:
Same as above, and same for the rest of the below `go.mod` files

Reply:
Same here